src/views/mixed-content/https-percentage-requests/index.js

```diff
@@ -0,0 +1,166 @@
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import MuiAlert from '@material-ui/lab/Alert';
+import React, {useState, useEffect} from 'react';
+import {api} from '../../../client';
+import {ChartContainer} from './elements';
+import {ResponsiveLine} from '@nivo/line';
+import {Typography, Snackbar} from '@material-ui/core';
+
+const HTTPSPercentageRequests = () => {
+  const [loading, setLoading] = useState(true);
+  const [data, setData] = useState([]);
+  const [snackOpen, setSnackOpen] = useState(false);
+
+  useEffect(() => {
+    api
+        .get('/api/mixed-content/https-percentage-requests')
```

Comment:
It seems like you have similar functionality in other files. Is it possible to put this into a common utils? Fine to do this in a separate PR.

Reply:
Sure, I converted this into a GitHub issue.